### PR TITLE
Landing page, calendar updates + pledge resources tab

### PR DESCRIPTION
**Pull request by tahiragrewal**
- made a comment
- added a comment
- Landing page and calendar updated, pledge resources created

### CODE REVIEW

File: src/Components/Portal/Tabs/Calendar/PledgeCalendar.jsx [hunk 1]
```diff
@@ -90,8 +90,50 @@ class PledgeCalendar extends React.Component {
         let date = new Date(Date.parse(event["Date"]));
         let type = event["Type"];
         let mand = event["Mandatory?"];
+        let group = event["Group"];
         let desc = event["Description"];
 
+        let eventStyle = {};
+        // Rush, Social, Pledge Task, Briefing, Capstone, Professional
+        if (type == 'Rush')
```
Comment (al3xisrobles) [inline src/Components/Portal/Tabs/Calendar/PledgeCalendar.jsx:98]: Ok bro 😭 use a switch statement instead

File: src/Components/Portal/Tabs/Calendar/PledgeCalendar.jsx [hunk 2]
```diff
@@ -263,6 +313,34 @@ class PledgeCalendar extends React.Component {
     return classes.filter(Boolean).join(" ");
   }
 
+  eventStyleGetter = (event) => {
+    let newStyle = {};
+    // Rush, Social, Pledge Task, Briefing, Capstone, Professional
+    // blue, yellow,orange, pink, green, purple
+    if (event.type === 'Rush') {
```
Comment (al3xisrobles) [inline src/Components/Portal/Tabs/Calendar/PledgeCalendar.jsx:320]: Again, use a switch

**Review by al3xisrobles — CHANGES_REQUESTED** [review]
Just fix those two switch statement things and re-PR for approval

File: src/Assets/Images/Exec/alice.jpeg
Comment (stevenewald) [inline src/Assets/Images/Exec/alice.jpeg:1]: These images should be downsized. They do not need to be this high res when they're displayed than 100x100px on the site

File: src/Components/Portal/Tabs/Calendar/PledgeCalendar.jsx [hunk 3]
```diff
@@ -90,8 +90,50 @@ class PledgeCalendar extends React.Component {
...
+            {
+              if (type == 'Briefing')
+              {
+                eventStyle = {backgroundColor:'pink'};
+              }
+              else
+              {
+                if (type == 'Capstone')
+                {
+                  eventStyle = {backgroundColor:'green'};
```
Comment (stevenewald) [inline src/Components/Portal/Tabs/Calendar/PledgeCalendar.jsx:124]: bro 💀💀💀

Comment (stevenewald) [reply src/Components/Portal/Tabs/Calendar/PledgeCalendar.jsx:124]: Also you should use === for js strings

**Review by stevenewald — CHANGES_REQUESTED** [review]

File: src/Assets/Images/Exec/alice.jpeg
Comment (al3xisrobles) [reply src/Assets/Images/Exec/alice.jpeg:1]: 🙌

**Comment by al3xisrobles** [comment]
Tahira's busy right now but I want to merge this so making fixes on my own and pushing to main.

**Comment by al3xisrobles** [comment]
Ok I didn't realize u set PR-only (no push) permissions loll but whatever will open in new PR

**Pull request closed without merging**